background/lib/fixed-point.ts

```diff
@@ -223,6 +223,8 @@ export function fixedPointNumberToString(
  * fixed point precision is used.
  *
  * This function is best used as the last step after any computations are done.
+ * 
+ * NOTE: This function TRUNCATES to desiredDecimals but does not ROUND.
```

Comment:
Correct me if I'm wrong, but doesn't truncation inherently round down? What do you mean it doesn't round?

Reply:
Truncation rounds down but does not round up. I have edited the comment to include this.